### DISCUSSION TITLE
How do I change the icons?

### DISCUSSION THREAD

**Discussion by QueenUlyana** [d2phap/ImageGlass#1603] [Q&A] [answered]
How to change the icons that were installed after installing ImageGlass. I either want to return the previous basic icons or put other icons.
![2023-06-23_184341](https://github.com/d2phap/ImageGlass/assets/83131824/d7da8fb4-1b68-400c-8f08-b0542ec52a3f)

**Comment by d2phap** [comment]
### 1. To delete ImageGlass file type icons
- You can open ImageGlass installed folder, delete all icons file in `Ext-Icons` folder

### 2. To change the icons
- Download the new icons, for example: https://imageglass.org/theme/jellies-icon-set-h-one-design-48
- Extract and copy all `.ico` files to the `Ext-Icons` folder

### 3. Apply changes
- Open ImageGlass Settings > File type associations:
  - Click `Unregister extensions` button if you want to remove all icons and file type association
  - Click `Set as default photo viewer` if you want to remove all icons and file type association, then register them again with new icons

**Comment by QueenUlyana** [comment]
> ### 1. To delete ImageGlass file type icons
> * You can open ImageGlass installed folder, delete all icons file in `Ext-Icons` folder
> 
> ### 2. To change the icons
> * Download the new icons, for example: https://imageglass.org/theme/jellies-icon-set-h-one-design-48
> * Extract and copy all `.ico` files to the `Ext-Icons` folder
> 
> ### 3. Apply changes
> * Open ImageGlass Settings > File type associations:
>   
>   * Click `Unregister extensions` button if you want to remove all icons and file type association
>   * Click `Set as default photo viewer` if you want to remove all icons and file type association, then register them again with new icons

I tried that, but it didn't work. And you can not do so in the beta, there are all the settings in a text file. It's a total mess.

**Reply by d2phap** [reply, marked as answer]
The above steps are for the stable release, not the beta. You should have mentioned what version you're using to get better answer and not to waste people's time.

For the steps that works with beta version, you can read the release notes of v9.0 beta 1.
